lib/pxutil.js

```diff
@@ -185,6 +186,10 @@ function extractIP(config, req) {
     return req.ip;
 }
 
+function readJsonFile(filepath) {
+    return JSON.parse(fs.readFileSync(filepath, 'utf8'));
```

Comment:
you should probably wrap this with `try - catch`

Reply:
I'm wrapping this outside the call, I want to control exceptions there.